lib/locomotive/steam/services/site_finder_service.rb

```diff
@@ -6,7 +6,7 @@ class SiteFinderService
       attr_accessor_initialize :repository, :request
 
       def find
-        repository.by_domain(request.host)
+        repository.by_domain(request.host) if request
```

Comment:
alright, I was able to remove the `if request` just by changing the way we build the real host (https://github.com/locomotivecms/steam/commit/5b6b6100c8513bede4f9902378476f54afa3712b).